src/types.ts

```diff
@@ -16,6 +16,7 @@ type AstMetadataApi = {
   type?: string;
   name?: string;
   object: string;
+  astNodeType: string; // "MemberExpression" | "CallExpression" | "NewExpression"
```

Comment:
Should we use `"MemberExpression" | "CallExpression" | "NewExpression"`? I feel like that's strictly better than `string`